Terminal.Gui/ConsoleDrivers/ConsoleDriver.cs

```diff
@@ -315,12 +315,31 @@ public void ClearContents ()
                 {
                     Contents [row, c] = new Cell
                     {
-                        Rune = (Rune)' ', 
-                        Attribute = new Attribute (Color.White, Color.Black), 
+                        Rune = (Rune)' ',
+                        Attribute = new Attribute (Color.White, Color.Black),
                         IsDirty = true
                     };
-                    _dirtyLines [row] = true;
                 }
+                _dirtyLines [row] = true;
+            }
+        }
+    }
+
+    /// <summary>
+    /// Sets <see cref="Contents"/> as dirty for situations where views
+    /// don't need layout and redrawing, but just refresh the screen.
+    /// </summary>
+    public void SetContentsAsDirty ()
```

Comment:
1) What does this have to do with Ctrl-Z? 
2) I think this would be better implemented as 

```cs
public bool ForceFullUpdate {get;set;}
```

Then in `Update`:

```cs
      if (ForceFullUpdate || !Contents [row, col].IsDirty)
...

ForceFullUpdate = false;
```

Reply:
And can't it be made `private` since it's only called from a derived class?

Reply:
> 1. What does this have to do with Ctrl-Z?

This is related with the issue #3525 and was intended to use in the `Suspend` method, so it have to do with Ctrl-Z and I leveraged this branch to implement.

> 2. I think this would be better implemented as
> 
> ```cs
> public bool ForceFullUpdate {get;set;}
> ```
> 
> Then in `Update`:
> 
> ```cs
>       if (ForceFullUpdate || !Contents [row, col].IsDirty)
> ...
> 
> ForceFullUpdate = false;
> ```

I don't agree because if it use a property I have to include that code on two derived class, `CursesDriver` and `NetDriver`. Using a method on the `ConsoleDriver` it's only implemented once. If you don't like the method name, then you may suggest another name.

Reply:
> And can't it be made `private` since it's only called from a derived class?

No, because it is inaccessible in derived classes if I change to private.

Reply:
Sounds good. I appreciate the explanation; makes sense.